pkg/cli/admin/ocpcertificates/certgraphanalysis/certkeypair.go

```diff
@@ -0,0 +1,213 @@
+package certgraphanalysis
+
+import (
+	"crypto/ecdsa"
+	"crypto/rsa"
+	"crypto/x509"
+	"fmt"
+	"os"
+
+	"github.com/openshift/oc/pkg/cli/admin/ocpcertificates/certgraphapi"
+	"k8s.io/apimachinery/pkg/util/duration"
+)
+
+func toCertKeyPair(certificate *x509.Certificate) (*certgraphapi.CertKeyPair, error) {
+	ret := &certgraphapi.CertKeyPair{
+		Name: fmt.Sprintf("%v::%v", certificate.Subject.CommonName, certificate.SerialNumber),
```

Comment:
`Subject` has a `String()` method that might work better than just the common name